installers/docker/files/init-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# ----------------------------------------------------------------------------------
+#
+# Generate required artifacts for ballerina-runtime docker image creation
+set -e
+
+USER_ID=${1}
+MODULE_DIR=${2}
+TEMP_DIR=tmp
+BAL_PROJECT="$MODULE_DIR"_proj
+B7a_EXECUTABLE=/usr/lib/ballerina/ballerina-1.0.3/bin/ballerina
```

Comment:
Do we need this absolute path? The ballerina image already has set this in the PATH variable.

Reply:
If it is available in the path, then we can remove it. Resolved